### ISSUE TITLE
Bug: Korean Character problem

### ISSUE THREAD

**Issue by joniel** [facebook/lexical#6913]
Lexical version:

## Steps To Reproduce

1. https://youtu.be/mGDYKGO0kLo
2. forgive me my lackness of english ... 
3. I hope to show that i cannot input korean character without first blank charcter. 
4. it's same problem in mdx-editor too. not ckeditor. 
5. i think about css3 too... ` user-select: text;`

Link to code example:

**Comment by etrepum** [comment]
I can't reproduce this one on macOS + Chrome

**Comment by joniel** [comment]
> I can't reproduce this one on macOS + Chrome

Thanks. but it burst on "Ubuntu + Chrome" : )
macOS is ok. no problems.

**Comment by etrepum** [comment]
Yes, I saw that you said it doesn't work in Ubuntu + Chrome, but you didn't say that you tested it on other platforms. I was just adding more information, I don't have an Ubuntu machine or VM ready to easily test or debug there.

**Comment by mayrang** [comment]
Tested on Ubuntu 24.04 + Chrome 150 + fcitx-hangul (Lexical v0.22.0, current main). Korean input works without needing a leading blank character. The original report didn't specify a Lexical version, so this may have been fixed in a subsequent release. Closing as not reproducible — feel free to reopen if you can still reproduce on the latest version.
